specification/07-theory_operations.adoc

```diff
@@ -431,4 +431,98 @@ TSM ->> VMM: [COVH] - spdm_covh_connect_device()
 
 === Interface Assignment
 
+Once both the SPDM session and the IDE link are secured and established, the
+host supervisor domain manager may directly assign a TDI to a TVM, through the
+`COVH` interface. This is a four steps process:
+
+1. The host supervisor domain manager initiates the interface assignment flow by
+   having the TSM move the TDI into the TDISP `LOCKED` state.
+2. The TVM verifies and accepts the locked TDI into its TCB.
+3. The host supervisor domain manager asks the TSM to move the TDI to the TDISP
+   `RUNNING` state.
+4. The TVM verifies that the TDI is in the TDISP `RUNNING` state and starts
+   using it.
+
+The next section describe in more details what step 2 covers, i.e. which steps
+a TVM should follow in order to be able to verify a locked TDI and accept or
+reject it into its TCB.
+
+==== TDI Acceptation
+
+It is the TVM responsibility to accept or reject the assigned TDI into its
+TCB, and to explicitly notify both the TSM and the host supervisor domain
+manager about its decision. The TVM should check for the following security
+attributes before being able to decide whether or not it can safely add a TDI
+into its TCB:
+
+1. **SPDM session establishment**: A secured SPDM session must be established
+   between the TDI’s DSM and the TSM. TVM verifies that attribute from the TSM,
+   through the `sbi_covg_get_device_link()` `COVG` ABI.
+2. **IDE link**: The PCIe physical link between the Root Port and the physical
+   device must be confidentiality and integrity protected through IDE. As for
+   the SPDM session, the TVM calls into the `sbi_covg_get_device_link()` `COVG`
+   ABI to verify that attribute from the TSM.
+3. **Firmware update policy**: The TVM must verify that the device firmware
+   update security attributes are acceptable, by calling into
+   `sbi_covg_get_device_update_attrs()`. The returned set of attributes,
+   including the firmware update policy, lets the TVM check if runtime firmware
+   update is allowed for the device, and under which conditions. Depending on
+   the TVM’s own policy, this may be acceptable or not.
+4. **TDI state**: Before accepting a TDI into its TCB, a TVM must verify that
+   its configuration is immutable, and in particular that the host can not
+   modify it without having all in-flight transactions being discarded. TEE-IO
+   capable physical devices follow the TDISP specification and can guarantee
+   that immutability state once the TDI has been moved to the TDISP LOCKED
+   state. The transition from TDISP `UNLOCKED` to `LOCKED`, although triggered
+   by the host supervisor domain manager, is done by the TSM. As such, the TVM
+   can query the TSM for the TDI state through the
+   `sbi_covg_get_interface_state()` `COVG` ABI. A TVM must not accept a TDI if
+   it’s in any other TDISP state than LOCKED.
+5. **TDI identity**: Verifying that the TDI is in an immutable state across a
```

Comment:
Suggest to use `TDI identity and measurement` to make it explicit.

Identity usually only means certificate.